stream_with_sentences.py

```diff
@@ -9,12 +9,51 @@
 import time
 import curses
 import textwrap
+import sys
+import getopt
+import locale
+
+TRANSLATION_LANGUAGE = ''
+WAVE_FILE = ''
+MODEL_DIR = ''
 
+def usage():
+    print "Usage:" + sys.argv[0] + " --lang <en|de|es|fr|pl> --file </path/to/.wav> --models </path/to/models>"
 
-MODEL = "models/output_graph.pbmm"
-ALPHABET = "models/alphabet.txt"
-LM = "models/lm.binary"
-TRIE = "models/trie"
+try:
+    opts, args = getopt.getopt(sys.argv[1:], 'l:f:m:h', ['lang=', 'file=', 'models=', 'help'])
+except getopt.GetoptError:
+    usage()
+    sys.exit(2)
+
+for opt, arg in opts:
+    if opt in ('-h', '--help'):
+        usage()
+        sys.exit(2)
+    elif opt in ('-l', '--lang'):
+        TRANSLATION_LANGUAGE = arg
+    elif opt in ('-f', '--file'):
+        WAVE_FILE = arg
+    elif opt in ('-m', '--models'):
+        MODEL_DIR = arg
```

Comment:
Good idea to make this a parameter for the script. I did notice one other place we need to use MODEL_DIR in order for the jamspell spell correction to work. It needs to load the 'en.bin' language model, which the script assumes to be in the current directory. Jamspell helpfully doesn't emit an error when it can't load the model and will instead just not correct spelling. :)